src/plugins/intel_gpu/src/runtime/engine.cpp

```diff
@@ -254,9 +254,19 @@ std::shared_ptr<cldnn::engine> engine::create(engine_types engine_type,
                                               const engine_configuration& configuration,
                                               const InferenceEngine::ITaskExecutor::Ptr task_executor) {
     device_query query(engine_type, runtime_type);
-    device::ptr default_device = query.get_available_devices().begin()->second;
+    auto devices = query.get_available_devices();
 
-    return engine::create(engine_type, runtime_type, default_device, configuration, task_executor);
+    auto iter = devices.find(debug_configuration::get_device_id());
+    auto& device = iter != devices.end() ? iter->second : devices.begin()->second;
+
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_IF(debug_config->verbose >= 1) {
+        const auto& info = device->get_info();
+        GPU_DEBUG_COUT << "Selected Device: " << info.dev_name << std::endl;
+        GPU_DEBUG_COUT << "Device support immad: " << (info.supports_immad ? "YES" : "NO") << std::endl;
+    }
```

Comment:
move this print to inner `::create` function

Reply:
Done.